src/core/routing/transfer-route.ts

```
@@ -0,0 +1,685 @@
+        {
+          stationId: originId,
+          lineId: lineId,
+          isTransfer: false,
+        },
+      ],
+    });
+  });
+
+  // BFS loop
+  while (queue.length > 0) {
+    const currentState = queue.shift()!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)